subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -797,8 +787,12 @@ static void isr_rx(void *param)
 			iso_rx_sched();
 #endif /* CONFIG_BT_CTLR_LOW_LAT_ULL */
 
-			/* Increment burst number */
 			cis_lll->rx.bn_curr++;
+			if ((cis_lll->rx.bn_curr > cis_lll->rx.bn) &&
+			    ((cis_lll->rx.payload_count / cis_lll->rx.bn) < cis_lll->event_count)) {
```

Comment:
The comparison between `cis_lll->rx.payload_count / cis_lll->rx.bn` and `event_count` that happens multiple places, isn't something that I understand. Can you elaborate on how event count and `cis_lll->rx.payload_count / cis_lll->rx.bn` relate? 

Should/could the comparison (or at least the `cis_lll->rx.payload_count / cis_lll->rx.bn` calculation) be put in a named local variable or a function?

Reply:
The comparison here relates to the text in the spec.:
```
The burst of payloads associated with the
CIS event where cisEventCounter has the value E shall consist of payloads
with cisPayloadCounter between E × BN and (E + 1) × BN - 1.
```

Regarding refactoring, it will be addressed in #63152.